src/components/count-badge/CountBadge.scss

```diff
@@ -35,7 +35,7 @@
     height: 16px;
     line-height: 16px;
     min-width: 16px;
-    padding: 0 2px;
+    padding: 0 4px;
```

Comment:
we'll soon start coalescing on a 4-pixel grid for our sizings, so this component is now more aligned on that